app/scripts/lib/setupSentry.js

```diff
@@ -115,8 +115,19 @@ function getTracesSampleRate(sentryTarget) {
 
   const flags = getManifestFlags();
 
+  // Grab the tracesSampleRate that may have come in from a git message
+  // 0 is a valid value, so must explicitly check for undefined
+  if (flags.sentry?.tracesSampleRate !== undefined) {
+    return flags.sentry.tracesSampleRate;
+  }
+
   if (flags.circleci) {
-    return 0.003;
+    // Report very frequently on develop branch, and never on other branches
+    // (Unless you do a [flags.sentry.tracesSampleRate: x.xx] override)
+    if (flags.circleci.branch === 'develop') {
+      return 0.03;
```

Comment:
Minor, is it worth some constants for all these values for easier maintenance? Such as `TRACES_SAMPLE_RATE_CIRCLE_CI_DEVELOP`, `TRACES_SAMPLE_RATE_CIRCLE_CI_OTHER`, `TRACES_SAMPLE_RATE_DEBUG` etc?